sourcemap/elasticsearch.go

```diff
@@ -65,7 +65,8 @@ func (e *smapElasticsearch) runESQuery(body map[string]interface{}) (*es.SearchR
 	e.mu.Lock()
 	defer e.mu.Unlock()
 	for _, client := range e.clients {
-		_, result, err = client.Connection.SearchURIWithBody(e.index, "", nil, body)
+		_, result, err = client.Connection.SearchURIWithBody(e.index, "",
+			map[string]string{"rest_total_hits_as_int": "true"}, body)
```

Comment:
This parameter should be removed after elastic/beats#9735 makes it back through a beats update